### ISSUE TITLE
Solution Imp - Plan & Schedule Go Live

### ISSUE THREAD

**Issue by rhood23699** [CDCgov/prime-field-teams#132]
**Temporary New Daily Process:**
- iPatientcare will produce a daily CSV file by running their stored procedure and copying the results with Headers to a CSV file stored in the /out/directory.

- A6 will:
  - From the /Public Health/VBScript directory, A6 will run "Reddy-Send.bat"
  - A6 will review the .REP and .TXT files for errors.

Note, a new Issue will be created when we begin development of the 100% automation of the Reddy process.

**Potential Final Daily Process:**
- iPatientcare will automate the execution and creation of the CSV file.
- A6 will create a scheduled task to execute "Reddy-Send.bat" daily.
- Dr. Reddy's office will be responsible for reviewing the error files.

**Comment by rhood23699** [comment]
8/10/21 - Rick will work with Vinit on Wednesday to:
  - Vinit will create a results file from 7/23 thru 8/10.  Rick will extract and send approximately 20 Rows via the VBScript to RS Prod for Joel's testing.
  - Rick will clean and save the entire 7/23 thru 8/10 file.  Target date to GO LIVE and send this file is 8/13/2021.

8/10/21 - Pipeline Team was notified and agreed to PUSH Reddy changes on Thursday.  The 20 results will be sent to LADOH on Thursday if they have their blocking in place with the hopes to GO LIVE on Friday.  If not, we'll send the 20 results on Friday for validation which would PUSH GO LIVE to Monday.

**Comment by rhood23699** [comment]
Latest changes to Prod/Test will be PUSH'd on Tuesday, 17th at 11:30a.  We'll resend the 20+ results to LADOH.  If they stated the results are good, we'll send Reddy results from 7/23/21 thru 8/11/21 as soon as LADOH says they are ready.

**Comment by rhood23699** [comment]
LADOH is unblocking Dr. Reddy results.  They will contact us when they are ready to receive the backlog of results.

Joel is working on the PUSH to allow results to be sent to HHS Protect.

Tentative GO LIVE is tomorrow, Wednesday the 18th.

**Comment by rhood23699** [comment]
A summary of the Reddy setup is at the link https://docs.google.com/document/d/1Rc7fYAPiXKAjjqThUA7tc3NCJdEbE1HrQf8dsdHbzUQ/edit, but note, see the Description of this issue for a summary of the temporary daily process currently being used.

**Comment by rhood23699** [comment]
Updated the VBScript to transform the DOB to m/d/yyyy h:mm, and Date Collected and Date Resulted to m/d/yyyy on the raw/original CSV file generated by iPatientcare to alleviate the need to edit the CSV file in EXCEL.  I updated the Temporary Daily Process in the Description of this Issue.
